src/control-plane/backend/lambda/api/service/quicksight/reporting-utils.ts

```diff
@@ -507,6 +507,23 @@ export function getFunnelTableVisualDef(visualId: string, viewName: string, even
             DataSetIdentifier: viewName,
             ColumnName: 'rate',
           },
+          FormatConfiguration: {
+            FormatConfiguration: {
```

Comment:
why has the duplicated field name?

Reply:
Although it looks strange, it is correct.